Sources/MapboxNavigation/Resources/Base.lproj/Navigation.storyboard

```diff
@@ -141,28 +140,27 @@
+                            <constraint firstItem="ekW-06-trL" firstAttribute="leading" secondItem="IDF-XF-2Ta" secondAttribute="leading" id="TeT-ls-Ns4"/>
+                            <constraint firstItem="ekW-06-trL" firstAttribute="trailing" secondItem="IDF-XF-2Ta" secondAttribute="trailing" id="UUM-FL-FrV"/>
+                            <constraint firstAttribute="trailing" secondItem="TEs-GQ-xWN" secondAttribute="trailing" id="Vg9-Xi-JtV"/>
                             <constraint firstItem="unQ-4I-UPo" firstAttribute="leading" secondItem="IDF-XF-2Ta" secondAttribute="leading" id="WJ9-OZ-x1y"/>
                             <constraint firstAttribute="bottom" secondItem="ekW-06-trL" secondAttribute="bottom" id="f1v-ju-qen"/>
                             <constraint firstItem="unQ-4I-UPo" firstAttribute="top" secondItem="IDF-XF-2Ta" secondAttribute="top" id="gVo-YH-zU7"/>
-                            <constraint firstItem="evE-dd-pMr" firstAttribute="leading" secondItem="P8U-kf-mkb" secondAttribute="leading" id="k0v-jB-TbO"/>
-                            <constraint firstItem="TEs-GQ-xWN" firstAttribute="leading" secondItem="P8U-kf-mkb" secondAttribute="leading" id="lxR-dT-lxi"/>
-                            <constraint firstItem="4jh-bR-wXL" firstAttribute="leading" secondItem="P8U-kf-mkb" secondAttribute="leading" id="muP-uc-2iF"/>
+                            <constraint firstItem="evE-dd-pMr" firstAttribute="leading" secondItem="IDF-XF-2Ta" secondAttribute="leading" id="k0v-jB-TbO"/>
+                            <constraint firstItem="TEs-GQ-xWN" firstAttribute="leading" secondItem="IDF-XF-2Ta" secondAttribute="leading" id="lxR-dT-lxi"/>
+                            <constraint firstItem="4jh-bR-wXL" firstAttribute="leading" secondItem="IDF-XF-2Ta" secondAttribute="leading" id="muP-uc-2iF"/>
                             <constraint firstItem="4jh-bR-wXL" firstAttribute="top" secondItem="IDF-XF-2Ta" secondAttribute="top" constant="16" id="rx6-4c-WD7"/>
-                            <constraint firstItem="P8U-kf-mkb" firstAttribute="trailing" secondItem="4jh-bR-wXL" secondAttribute="trailing" id="s63-sP-XUC"/>
+                            <constraint firstAttribute="trailing" secondItem="4jh-bR-wXL" secondAttribute="trailing" id="s63-sP-XUC"/>
                             <constraint firstItem="evE-dd-pMr" firstAttribute="top" secondItem="4jh-bR-wXL" secondAttribute="bottom" constant="16" id="vRY-cm-7kg"/>
-                            <constraint firstItem="D6B-Ff-n7b" firstAttribute="bottom" secondItem="P8U-kf-mkb" secondAttribute="bottom" id="xqf-EL-Wnt"/>
+                            <constraint firstItem="D6B-Ff-n7b" firstAttribute="bottom" secondItem="IDF-XF-2Ta" secondAttribute="bottom" id="xqf-EL-Wnt"/>
                             <constraint firstItem="ekW-06-trL" firstAttribute="top" secondItem="D6B-Ff-n7b" secondAttribute="top" id="zG7-CU-Ygb"/>
```

Comment:
Looks like there is a regression here. `End Navigation` button placement is no longer correct:
Before:
![Simulator Screen Shot - iPhone 13 - 2022-12-05 at 09 39 28](https://user-images.githubusercontent.com/1496498/205705470-f9b8d869-ec52-4bc1-b40c-3117adb70c70.png)

After:
![Simulator Screen Shot - iPhone 13 - 2022-12-05 at 09 37 51](https://user-images.githubusercontent.com/1496498/205705511-a11d5451-ecef-45b9-8850-4ff3543135af.png)

Please fix it before landing this PR, otherwise looks like everything else is fine.